### PR TITLE
Feature/forms

### PR DESCRIPTION
**Pull request by JuanSeBestia**
feat: generic forms, please read README.md

feat: view new patient page

image

### CODE REVIEW

File: src/app/pages/new-patient/new-patient.component.ts [hunk 1]
```diff
@@ -1,15 +1,69 @@
...
 })
 export class NewPatientComponent implements OnInit {
+  inputs: InputBase[] = [
+    // Instance Dropdown with class
+    new InputDropdown({
+      key: 'tipo_documento',
+      label: 'Tipo de documento',
+      options: [
+        { value: 'CEDULA', text: 'Cédula' },
+        { value: 'CEDULA_EXTRANGERA', text: 'Cédula extrangera' },
```
Comment (KaterineMoon) [inline src/app/pages/new-patient/new-patient.component.ts:21]: Juani.... Extranjera*

File: src/app/pages/new-patient/new-patient.component.ts [hunk 2]
```diff
@@ -1,15 +1,69 @@
...
+        { value: 'PASAPORTE', text: 'Pasaport' },
```
Comment (KaterineMoon) [inline src/app/pages/new-patient/new-patient.component.ts:22]: Pasaporte*

File: src/app/pages/new-patient/new-patient.component.ts (hunk 1 above)
Comment (JuanSeBestia) [reply src/app/pages/new-patient/new-patient.component.ts:21]: Done

File: src/app/pages/new-patient/new-patient.component.ts (hunk 2 above)
Comment (JuanSeBestia) [reply src/app/pages/new-patient/new-patient.component.ts]: Done

**Pull request merged**